tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Azure SDK Tools Contribution Guidelines
```

Comment:
It is worth calling out that we allow pretty much any language for the tools.

Reply:
Added in Line #477